contrib/packages/rpm/el7/SPECS/tigervnc.spec

```diff
@@ -255,6 +255,7 @@ fi
 %doc %{_docdir}/%{name}/README.rst
 %{_bindir}/vncviewer
 %{_datadir}/applications/*
+%{_datadir}/metainfo/*
```

Comment:
Please squash this in to the original commit. Less confusion that way.

Reply:
ACK. Should be integrated now.